kale/embed/mpca.py

```diff
@@ -74,7 +74,7 @@ class MPCA(BaseEstimator, TransformerMixin):
         n_components (int): Number of components to keep. Applies only when return_vector=True. Defaults to None.
 
     Attributes:
-        proj_mats (list of array-like): A list of transposed projection matrices, shapes (P_1, I_1), ...,
+        proj_mats (list of arraies): A list of transposed projection matrices, shapes (P_1, I_1), ...,
```

Comment:
Typo: "arraies" should "arrays", see https://uk.mathworks.com/help/matlab/math/multidimensional-arrays.html